.gitignore

```diff
@@ -6,7 +6,8 @@ output
 /byoqm/main.html
 /metrics/cpd/
 /documentation/
-
+/byoqm/util/.ignore_paths.txt
+/byoqm/util/.ignore_extensions.txt
```

Comment:
Tools I know with ignore files have the naming convention `.ignore`, e.g.
- `.gitignore`
- `.dockerignore`

Maybe we should call it `.moduignore`?, also we could have both full paths and extensions in the same file